hw/ip/entropy_src/dv/entropy_src_sim_cfg.hjson

```diff
@@ -32,7 +32,7 @@
                 "{proj_root}/hw/dv/tools/dvsim/tests/intr_test.hjson",
                 "{proj_root}/hw/dv/tools/dvsim/tests/alert_test.hjson",
                 "{proj_root}/hw/dv/tools/dvsim/tests/tl_access_tests.hjson",
-                "{proj_root}/hw/dv/tools/dvsim/tests/stress_tests.hjson"]
+                "{proj_root}/hw/dv/tools/dvsim/tests/stress_all_test.hjson"]
```

Comment:
can you add a TODO here to switch back to `stress_tests` later?

Reply:
Thanks weicai